### PR TITLE
Upgrade Wagtail inventory to 1.6

### PR DESCRIPTION
**Pull request by kevinhowbrook**
# Description

Based off https://github.com/mozilla/foundation.mozilla.org/pull/9894

This PR upgrades wagtail inventory in preparation for the main wagtail upgrade https://github.com/mozilla/foundation.mozilla.org/issues/9674

Link to sample test page:
Related PRs/issues: #9676 

# Checklist

**Tests**
- [x] Is the code I'm adding covered by tests? - existing tests

**Changes in Models:**
- [ ] ~~Did I update or add new fake data?~~ No
- [ ] ~~Did I squash my migration?~~ No migrations
- [ ] ~~Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?~~ No migrations

**Documentation:**
- [ ] ~~Is my code documented?~~ Not needed
- [ ] ~~Did I update the READMEs or wagtail documentation?~~ Not needed

### CODE REVIEW

**Review by tbrlpld — APPROVED** [review]

File: requirements.in [hunk 1]
```diff
@@ -29,8 +29,8 @@ wagtail==2.16.3
 wagtail-color-panel
 wagtail-factories
 wagtail-localize==1.1
-wagtail-localize-git==0.12.0
-wagtail-inventory
+wagtail-localize-git==0.13.0
+wagtail-inventory==1.6
```
Comment (tbrlpld) [inline requirements.in:33]: @kevinhowbrook Why are we pinning the version in the `requirements.in` file? Shouldn't this be resolved by `pip-compile` and locked in the `requirements.txt`?

Comment (kevinhowbrook) [reply requirements.in]: Thanks, localize-git was already pinned so I continued with that.
Have changed this though so both aren't pinned and are resolved with compile and lock

Comment (tbrlpld) [reply requirements.in]: Nice one. Thanks @kevinhowbrook.

**Review by tbrlpld — APPROVED** [review]

**Comment by tbrlpld** [comment]
@kevinhowbrook Can you merge these now, or is that still with me? Last info from @cdanfon was you should be able to merge now.

**Pull request merged**